src/core/util/prefixer.js

```diff
@@ -0,0 +1,63 @@
+angular
+  .module('material.core')
+  .config( function($provide) {
+    $provide.decorator('$mdUtil', ['$delegate', function ($delegate) {
+
+      // Inject the prefixer into our original $mdUtil service.
+      $delegate.prefixer = MdPrefixer;

```

Comment:
FYI: This comment is already outdated, since @ThomasBurleson had a discussion about it and figured out a better solution.